src/InputForm/InputForm.tsx

```diff
@@ -58,14 +167,38 @@ export const InputForm = (props?: InputFormProps) => {
             }
         }
 
+
```

Comment:
rzeczy pokomentowany bym wywalił o ile nie są potrzebne, wtedy można opis zostawić co to jest